mitodl/ocw-hugo-projects
# add basic course hugo config

This PR adds the basic Hugo config for course sites built using the `course` theme as well as an empty `ocw-studio` config

To test this PR:
 - Clone https://github.com/mitodl/ocw-hugo-themes
 - From `ocw-hugo-themes` directory, run `yarn install && npm run start:site:webpack`
 - Clone https://github.com/mitodl/ocw-to-hugo
 - Create markdown for any course using the instructions in the readme
 - From any course in your output directory, run:
 ```
hugo server --config /path/to/ocw-hugo-projects/ocw-course/config.yaml --themesDir /path/to/ocw-hugo-themes/
 ```

The course site should be spun up at http://localhost:1313
